### PR TITLE
Add get exports list

### PR DESCRIPTION
**Pull request by pthegner**
# Description

This PR add the call `impacts/exports`

## Issue

Issue link: BOT-1270

### CODE REVIEW

**Comment by dvdn** [comment]
Why not add also new forgotten routes '/_search' ?

**Comment by pthegner** [comment]
retest this please

**Comment by pthegner** [comment]
> 
> 
> Why not add also new forgotten routes '/_search' ?

Because it's a POST route

File: tests/features/list-export.feature [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        When I get "/impacts/exports"
+        Then the status code should be "200"
+        And the header "Content-Type" should be "application/json"
+        And the field "exports" should have a size of 2
+        And the field "exports.0.id" should be "7ffab230-3d48-4eea-aa2c-22f8680230b6"
+        And the field "exports.0.status" should be "done"
+        And the field "exports.0.created_at" should be "2014-04-02T23:52:12Z"
+        And the field "exports.0.process_start_date" should be "2014-04-02T23:55:12Z"
+        And the field "exports.0.start_date" should be "2014-03-01T00:00:00Z"
+        And the field "exports.0.end_date" should be "2014-04-01T00:00:00Z"
```
Comment (Fahmus) [inline tests/features/list-export.feature]: incomplete test:
And the field "exports.0.url" should be ??
/impact/exports returns 
`"exports":[
{
"created_at": "2014-04-02T23:52:12Z",
"end_date": "2014-04-01T00:00:00Z",
"id": "7ffab230-3d48-4eea-aa2c-22f8680230b6",
"process_start_date": "2014-04-02T23:55:12Z",
"start_date": "2014-03-01T00:00:00Z",
"status": "done",
"updated_at": "2014-04-02T23:52:12Z",
"url": null
}
, .....
]`

Comment (pthegner) [reply tests/features/list-export.feature]: not implemented now, what would you want to return here without the BOT-1285 or the BOT-1284 ?

File: tests/features/list-export.feature [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    Scenario: list without client in the header fails
+        Given I remove header "X-Customer-Id"
+        When I get "/impacts/exports"
+        Then the status code should be "400"
+
+    Scenario: list without client in the database fails
+        When I get "/impacts/exports"
+        Then the status code should be "404"
+
+    Scenario: if there is no channel the list is empty
```
Comment (dvdn) [inline tests/features/list-export.feature]:
```suggestion
    Scenario: if no export for a client, the list is empty
```

**Pull request merged**